src/test/java/org/llorllale/cactoos/matchers/InputHasContentTest.java

```diff
@@ -46,7 +46,7 @@
     @Test
     public void matchesInputContent() {
```

Comment:
`Test case for {@link org.llorllale.cactoos.matchers.InputHasContent}.` can be replaced with `Test case for {@link InputHasContent}.` The javadoc compiler will take the full path from the package declaration anyway.

Reply:
@ilyakharlamov fixed